src/main/java/ch/njol/skript/expressions/ExprInventoryChest.java

```diff
@@ -38,32 +38,31 @@
 @Name("Custom Chest Inventory")
 @Description("Returns a chest inventory with the given amount of rows and the name. Use the <a href=effects.html#EffOpenInventory>open inventory</a>effect to open it.")
 @Examples({"open chest inventory with 1 rows named \"test\"",
-           "set {_inv} to chest inventory with 1 row"})
+           "set {_inventory} to chest inventory with 1 row"})
 @Since("2.2-dev34")
-public class ExprNewInventory extends SimpleExpression<Inventory> {
+public class ExprInventoryChest extends SimpleExpression<Inventory> {
```

Comment:
What is InventoryChest? It should be ChestInventory

Reply:
It's InventoryChest so it's near the other inventory classes, I was already going to name it ChestInventory but InventoryChest was better.

Reply:
it isn't